docs/running-rules.md

```diff
@@ -9,42 +9,33 @@ import MoreHelp from "/src/components/MoreHelp"
 
-The following diagram displays the current number of Semgrep rules:
-<div className="lang-container" style={{marginBottom: '20px'}}>
-  <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
-</div>
-
-## Running Semgrep Registry rules
+## Running Semgrep Registry rules locally
 
-This section explains how to run [Semgrep Registry](https://semgrep.dev/explore) rules locally. See the following examples of how you may test Semgrep rules:
-
-1. Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code:
-    ```sh
-    semgrep --config=auto path/to/src
-    ```
-2. Explore the [Semgrep Registry](https://semgrep.dev/explore) and choose a rule.
-3. On the page of the rule, click **Run Locally**.
-4. Copy the code for local install, and then add the path to the source code you want to check:
-    <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" <span className="placeholder">PATH/TO/SRC</span></code></pre>
-5. Optional: Run registry rules simultaneously with local rules:
-   <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" --config=<span className="placeholder">PATH/TO/YML PATH/TO/SRC</span></code></pre>
+Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code (substitute the `PATH/TO/SRC` with path to your source code):
+<pre class="language-bash"><code>semgrep --config=auto <span className="placeholder">PATH/TO/SRC</span></code></pre>
 
-:::note
+:::info
 * `--config auto` sends your repository's project URL to [Semgrep Registry](https://semgrep.dev/r) to find rules configured for your repository and as a key for cached rule recommendations.
-* When Semgrep Registry is used, [usage metrics](../metrics) are collected by default.
 :::
 
-Add rulesets to Semgrep in CI scans using their "Add to Policy" button on Semgrep Community and Semgrep Team tiers.
+Explore the Semgrep Registry by following these steps:
+
+1. See the [Semgrep Registry](https://semgrep.dev/explore) and choose a rule.
+2. On the page of the rule, click **Run Locally**.
+3. Copy the code for local install, and then add the path to the source code you want to check in your terminal:
+    <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" <span className="placeholder">PATH/TO/SRC</span></code></pre>
+4. Optional: Run registry rules simultaneously with local rules:
+   <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" --config=<span className="placeholder">PATH/TO/YML PATH/TO/SRC</span></code></pre>
```

Comment:
maybe PATH/TO/REGISTRY/SRC and PATH/TO/LOCAL/YML just to clarify? I guess it doesn't matter the order, but it took me a minute since I was like "but they're both yaml..."

Reply:
@efortuna What about:

```bash
semgrep --config="RULESET-ID" --config=PATH/TO/RULE.YAML PATH/TO/SRC
```